src/style/global.css

```diff
@@ -1,8 +1,7 @@
 :root {
   --monospace: Menlo, Consolas, monospace;
   --monospace-font: 14px/1.5 var(--monospace);
-  --serif: "Source Serif Pro", "Iowan Old Style", "Apple Garamond", "Palatino Linotype", "Times New Roman",
-    "Droid Serif", Times, serif, "Apple Color Emoji", "Segoe UI Emoji", "Segoe UI Symbol";
+  --serif: "Source Serif 4", serif;
```

Comment:
```suggestion
  --serif: "Source Serif 4", Charter, "Bitstream Charter", "Sitka Text", Cambria, serif;
```

With this, when you set `globalStylesheets: []` or **when the user has no internet** (working offline, or can't connect to google fonts for some reason like corp. policy/firewall), you still get a decent font by default.

*source serif*
![source serif](https://github.com/user-attachments/assets/663ee574-8451-4b2e-90ab-4ea94e95db26)

*charter*
![charter](https://github.com/user-attachments/assets/9db3b01c-06d1-4efc-b9cb-27e53fef315c)

*times new roman*
![times](https://github.com/user-attachments/assets/d613a618-70df-4fbd-bfa2-e158724bf749)

Reply:
I was just following Google’s recommendation but I guess they don’t try very hard for when the font doesn’t load. I’m not sure we need to address this case really either, but if you feel we should, I’m inclined to just revert to what we had before and address it separately.